src/Sql/Sql.sqlproj

```diff
@@ -11,8 +11,6 @@
     <TargetFrameworkProfile />
   </PropertyGroup>
   <ItemGroup>
-    <Build Remove="dbo_future/**/*" />
-
```

Comment:
Do we know if this is fine? I'm slightly worried about this confusing DACPAC and future files being included.

Reply:
This is a remnant of an older process and was just missed with cleanup. I don't know of anything using "future".